app/pages/sections/catalogue-solutions/order-item/flat/ondemand/ui-tests/withSavedData.ui.test.js

```diff
@@ -154,6 +154,19 @@ test('should render the price table content', async (t) => {
     .expect(await extractInnerText(orderUnit)).eql(orderItem.itemUnit.description);
 });
 
+test('should render the delete button as not diasbled', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const button = Selector('[data-test-id="delete-button"] button');
+
+  await t
+    .expect(button.exists).ok()
+    .expect(await extractInnerText(button)).eql('Delete')
+    .expect(button.hasClass('nhsuk-button--secondary')).eql(true)
+    .expect(button.hasClass('nhsuk-button--disabled')).eql(false);
```

Comment:
👍

Reply:
Can you copy and paste this one to the ui withSavedData for flat-patient and flat-declartive too please